### PR TITLE
creating a bug in cacli1

### PR DESCRIPTION
**Pull request by Lakshmikanthwodeyar**

### CODE REVIEW

File: calc1.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+1)sum=`expr $n1 + $n2`
+   echo "Sum ="$sum;;
+# bug01 fixing the subtration issue
+2)sum=`expr $n1 - $n2`
+   echo "Sub = "$sum;;
+
+3)sum=`expr $n1 \* $n2`
+   echo "Mul = "$sum;;
+
+4)sum=`expr $n1 + $n2`
```
Comment (appars) [inline calc1.sh]: @Lakshmikanthwodeyar For division, looks incorrect, please correct -. otherwise looks good.

**Review by appars — CHANGES_REQUESTED** [review]
Please check my review comments.

**Review by Lakshmikanthwodeyar — COMMENTED** [review]
@appars I have made the changes. Plz review and approve the same.

**Review by appars — APPROVED** [review]
LGTM

**Pull request merged**